screens/CameraRecording.tsx

```diff
@@ -28,24 +28,38 @@ const CameraRecording: FunctionComponent = () => {
             <View style={Styles.bottomRow}>
                 <Button style={Styles.flip} onPress={() => {
                     setCameraType(cameraType == Camera.Constants.Type.front ? Camera.Constants.Type.back : Camera.Constants.Type.front)
-                }}> 
-                    <Text style={Styles.text}>Flip</Text> 
+                }}>
+                    <Text style={Styles.text}>Flip</Text>
                 </Button>
                 <Button style={Styles.takePicture} onPress={() => {
                     if (camera) {
-                        camera.takePictureAsync()
-                            .then(pic => {
-                                console.log(pic.uri);
+                        camera.takePictureAsync({quality: 0.3})
+                            .then(async pic => {
+                                 fs.uploadAsync('https://content.dropboxapi.com/2/files/upload', pic.uri, {
+                                    httpMethod: 'POST',
+                                    headers: { // TODO: Store access token securely
+                                        Authorization : "Bearer sl.An_PH4N_su65qxS7nv2vyqp7cPq_oTHP1Am6x15yJiwo0qzJqHxlguejoxKtrnhLL2XSxYBEqvpwGt3JdhsMvveg6K5mO6rMOHW3xf1wRAD-aHwxKXd95gCsOBuUEuYYDeEMA2F0qqE",
```

Comment:
is it fine to upload this to github, since our repo is public?

Reply:
It's actually not, Dropbox already disabled this token. We're still waiting for everyone to be available to have a meeting with DJ/Mitch to discuss using Firebase to authenticate and then have auth for Dropbox there as well.